thoth/report_processing/components/adviser.py

```diff
@@ -142,170 +138,235 @@ def _aggregate_thoth_results_from_ceph(
         for n, document_id in enumerate(adviser_ids):
             _LOGGER.debug(f"Analysis {document_id} n.{counter + 1}/{number_adviser_results}")
 
+            retrieved = False
             try:
                 document = adviser_store.retrieve_document(document_id)
-                files[document_id] = document
-
-                counter += 1
-
-                _LOGGER.info("Documents retrieved: %r", counter)
-
-                if limit_results:
-                    if counter == max_ids:
-                        return files, counter
+                retrieved = True
 
             except Exception as exception:
                 _LOGGER.exception(f"Exception during retrieval of adviser result {document_id}: {exception}")
                 continue
 
+            if retrieved:
```

Comment:
no need for `retrieved` variable and if check to be present if there is `continue` in exception handling